ush/python/pygfs/task/aero_analysis.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, config):
                 'npz_anl': self.config['LEVS'] - 1,
                 'AERO_WINDOW_BEGIN': _window_begin,
                 'AERO_WINDOW_LENGTH': f"PT{self.config['assim_freq']}H",
+                'aero_bkg_fhr': [int(t) for t in self.config['aero_bkg_times'].split(',')],
```

Comment:
`map` is a good way to do this without invoking `for` loops.
```suggestion
                'aero_bkg_fhr': map(int, self.config['aero_bkg_times'].split(',')),
```